spec/factories/orders.rb

```diff
@@ -3,6 +3,8 @@
 
 FactoryBot.define do
   factory :order_bid do
+    member { create(:member) }
```

Comment:
```suggestion
    member
```
и просто такой вариант вроде как должен подойти

Reply:
точно 👍